install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -179,6 +184,8 @@ containers:
         fieldPath: metadata.namespace
   - name: ISTIO_META_INTERCEPTION_MODE
     value: "{{ or (index .ObjectMeta.Annotations `sidecar.istio.io/interceptionMode`) .ProxyConfig.InterceptionMode.String }}"
+  - name: ISTIO_META_INBOUND_CAPTURE_PORTS
```

Comment:
If the annotation and values is called 'includeInboundPorts', let's call the meta INCLUDE_INBOUND_PORTS.
No need to use a different name.

Reply:
Will fix